javascript/example/DRACOLoader.js

```diff
@@ -15,6 +15,7 @@
 'use strict';
 
 THREE.DRACOLoader = function(manager) {
+    this.decoder = THREE.DRACOLoader.getDecoder();
```

Comment:
The only concern I have about having it as a member variable is that the `DracoModule` would be automatically instantiated when `THREE.DRACOLoader` is created even if there is no geometry decoded later on. I'm not sure if that's really a problem in practice though. I think it's ok for now, we can change it later if there are some problems with this approach. I'll merge it later today. Thanks!